src/userbase-server/db.js

```diff
@@ -197,62 +199,35 @@ const rollbackTransaction = async function (transaction) {
 
 exports.rollbackTransaction = rollbackTransaction
 
-const failedTxConditionCheckMsg = 'Make sure user has write permission to this db and the db id and hash are correct'
 const putTransaction = async function (transaction, userId, dbNameHash, databaseId) {
-  const user = await userController.getUserByUserId(userId)
+  const [user, database] = await Promise.all([
+    userController.getUserByUserId(userId),
```

Comment:
Why is it important that we check if the user exists? What's the worst that could happen if we don't do this? It seems that we might accept some transactions if a user is deleted, but I think that's okay. If the user change its username, we still use the same user_id, right? If so, that shouldn't be a problem (no data loss).

Reply:
>It seems that we might accept some transactions if a user is deleted

Yep, this is the worst that can happen, which technically is still possible even with this included. Including this just makes it simpler to prevent it in a circumstance like a user has 2 browsers open, deletes the user in Browser A, then tries to insert in Browser B.

Agreed this is ok to remove

Reply:
Cool, let's remove it then. This is a critical API, and the less we do the better (for both speed and reliability).